src/system/Zikula/Module/SettingsModule/Resources/views/Admin/modifyconfig.tpl

```diff
@@ -17,14 +17,16 @@
                     <div class="form-group">
                         <label class="col-lg-3 control-label" for="settings_{$varname}">{gt text='Site name'}</label>
                         <div class="col-lg-9">
-                            <input id="settings_{$varname}" type="text" class="form-control" name="settings[{$varname}]" value="{$modvars.ZConfig.$varname|safetext}" size="50" maxlength="100" />
+                            {if isset($modvars.ZConfig.$varname)}{assign var='varvalue' value=$modvars.ZConfig.$varname}{else}{assign var='varvalue' value=''}{/if}
+                            <input id="settings_{$varname}" type="text" class="form-control" name="settings[{$varname}]" value="{$varvalue|safetext}" size="50" maxlength="100" />
                         </div>
```

Comment:
this would be better with the `default` filter.

`{$varvalue|default:''|safetext}`

and so on.